### PR TITLE
add management route, controller action and service method to allow m…

### PR DESCRIPTION
**Pull request by segoddnja**

### CODE REVIEW

File: cg/src/domains/management/subjects/subjects.service.js [hunk 1]
```diff
@@ -65,7 +68,22 @@ class SubjectsService {
...
+      .join('subject_keys', 'subjects.id', '=', 'subject_keys.subject_id')
+      .select('personal_data')
+      .select('key')
+      .where({ subject_id: subjectId });
+
+    if (!data) {
+      throw new NotFound('Subject not found');
+    }
+    const decryptedData = decryptFromStorage(data.personal_data, data.key);
+    await recordAccessByController(subjectId);
```
Comment (sindelio) [inline cg/src/domains/management/subjects/subjects.service.js]: This records the access from a subject to it's own data, not the manager's access

Comment (segoddnja) [reply cg/src/domains/management/subjects/subjects.service.js]: Why the method name is `recordAccessByController` in this case?

Comment (sindelio) [reply cg/src/domains/management/subjects/subjects.service.js]: To differentiate between controller and processor actions

**Comment by sindelio** [comment]
Since it's done already, we should make use of it. In the future if the client don't want this feature, we can remove it. Please put it in a passing state @segoddnja and then we can merge

**Pull request merged**